src/pages/friend.js

```diff
@@ -29,14 +29,13 @@ function Friend() {
   const [notPaired, setNotPaired] = useState(false);
 
   useEffect(() => {
-    getFriend()
-      .then((response) => {
-        if (response.error) {
-          setNotPaired(true); 
-        } else {
-          setFriendData(response);
-        }
-      });
+    getFriend().then((response) => {
+      if (response.error) {
+        setNotPaired(true);
+      } else {
+        setFriendData(response);
+      }
+    });
```

Comment:
but `friendData` and `notPaired` are not defined there, only inside Friend component, and `getFriend` it's outside the component.

Reply:
I'm just waiting to merge the PR with the profile page because in that one I followed your advice and created a separate file just for getters, and I'll use it also in `friend.js` and `members.js`